cogs.csv

```diff
@@ -11,7 +11,7 @@ Encryptor,Generate and validify the strength of passwords.,"Tools, Fun",Kreusada
 Flags,Get the flag for a country,Tools,Kreusada
 Minifier,Minify your code with python minifier,Tools,Kreusada
 Mjolnir,Try to lift Thor's hammer,Fun,"Jojo, Kreusada"
-NameGenerator,Generate random names will optional gender arguments.,Fun,Kreusada
+NameGenerator,Generate random names with optional gender arguments.,Fun,Kreusada
 PingInvoke,r2d2? wall-e? Invoke ping by asking if your bot's there.,"Tools, Fun",Kreusada 
```

Comment:
Please could you remove the extra whitespace on the end of L15 so that the pre-commit checks don't fail